metagpt/prompts/sd_design.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+@Time    : 2023/8/18 09:51
+@Author  : stellahong
+@File    : __init__.py
+"""
+
+MODEL_SELECTION_PROMPT = """Please help me find a suitable model for painting in this scene.
+Model list will be given in the format like:
+'''
+model_name: model desc,
+'''
+
+you should select the model and tell me the model name. answer it in the form like Model: model_name || Domain:xxx
```

Comment:
It seems that doesn't contain a `Domain` info even with the given `{model_info}`.

Reply:
I request LLM to predict a domain using the model description.